frontend/docker-compose.yml

```diff
@@ -1,37 +1,43 @@
 version: "3.2"
 services:
   master:
-    image: trubudget/blockchain:latest
+    image: trubudget/blockchain:v1.27.0
```

Comment:
Why lock the TB version to 1.27? Also, why are the ports set to a fixed value & why is provisioning disabled in the docker-compose.yml?

Reply:
sorry didn't want to push this file is was for testing purposes